lib/main.dart

```diff
@@ -1,20 +1,30 @@
 import 'package:flutter/material.dart';
+import 'package:weather_app/screens/home_screen.dart';
+import 'package:weather_app/screens/login_page.dart';
 
 void main() {
-  runApp(const MainApp());
+  runApp(MyApp(
+    initialRoute: LoginPage.route,
+    routes: {
+      LoginPage.route: (context) => const LoginPage(),
+      HomeScreen.route: (context) => const HomeScreen(),
+    },
+  ));
 }
 
-class MainApp extends StatelessWidget {
-  const MainApp({super.key});
+class MyApp extends StatelessWidget {
+  const MyApp({super.key, required initialRoute, required routes});
```

Comment:
To access them, you need variables for them
```suggestion
  const MyApp({super.key, required this.initialRoute, required this.routes});

  final String initialRoute;
  final Map<String, WidgetBuilder> routes;
```